src/styles/utils/variables.scss

```diff
@@ -0,0 +1,18 @@
+$colorCell: #eee;
+
+$daySize: 100px;
+$countColomn: 7;
+$widthCall: $countColomn * ($daySize + 1px);
+$monthLength: 31;
+$lastDayToShow: 28;
+$fontSizeNumber: 0.3 * $daySize;
+$transformSize: 0.2 * $daySize;
+
+$days:
+  "mon" 0,
+  "tue" (($daySize + 1px) + 0.5px),
+  "wed" (($daySize + 1px) * 2 + 0.5px),
+  "thu" (($daySize + 1px) * 3 + 0.5px),
+  "fri" (($daySize + 1px) * 4 + 0.5px),
+  "sat" (($daySize + 1px) * 5 + 0.5px),
+  "sun" (($daySize + 1px) * 6 + 0.5px);
```

Comment:
Maybe it would be better to use a function for margin calculation. And this 0.5px intimidates me.